src/components/UserPosts/UserPosts.js

```diff
@@ -5,28 +5,28 @@ import Post from '../Post/Post.js'
 import * as gql from '../../queries/queries';
 import './UserPosts.css'
 import UserContext from '../UserProfile/UserContext.js';
+import Loading from '../Loading/Loading.js'
 
 // const dayjs = require('dayjs')
 // const LocalizedFormat = require('dayjs/plugin/localizedFormat')
 // dayjs.extend(LocalizedFormat)
 
 const UserPosts = () => {
-  
+
   const value = useContext(UserContext);
-  
+
   const [query, setQuery] = useState(1)
   const [clicked, setClicked] = useState(false)
   const { loading, error, data } = useQuery(gql.GET_USER_POSTS(value));
   const [deletePost] = useMutation(gql.DELETE_POST, {
     refetchQueries: [{ query: gql.GET_USER_POSTS(value) }],
   });
 
-
   useEffect(() => {
     setClicked(false)
   }, [data])
 
-  if (loading) return <p>Loading...</p>;
+  if (loading) return <Loading loading={loading} />;
```

Comment:
I appreciate you consistently using the loading component in each of our react components, Bobby

Reply:
It's really clear to follow